packages/local-environment/src/agent.ts

```diff
@@ -20,11 +35,11 @@ export interface Agent {
 }
 
 export enum AgentType {
-  Updater,
-  Relayer,
-  Processor,
-  Watcher,
-  Kathy,
+  Updater = 'updater',
```

Comment:
Is there a reason you set values to the enum members here? I would roll this back unless there's an explicit thing this is fixing.

Reply:
EDIT: Well I see this enabled you to remove `agentTypeToString` which is an improvement I think

Reply:
Would maybe ask @prestwich if he has any hot-takes around the philosophically best way to handle this, I think adding literal values to the enum is a good solution right now.

Reply:
Doing it the Xiangan's way is definitely correct. Writing this I was not aware of TS' way of assigning strings to enums